FrontEnd/src/pages/AdminPage/UserProfilesTable/UserTable.jsx

```diff
@@ -66,6 +68,60 @@ function UserTable() {
             <CaretDownOutlined className={css['icon']} />
         );
     };
+    const handleSearch = (selectedKeys, confirm, dataIndex) => {
+        confirm();
+        setSearchText(selectedKeys[0]);
+        setSearchedColumn(dataIndex);
+  };
+    const handleReset = (clearFilters) => {
+        clearFilters();
+        setSearchText('');
+    };
+    const getColumnSearchProps = (dataIndex) => ({
+        filterDropdown: ({setSelectedKeys, selectedKeys, confirm, clearFilters}) => (
+            <div className={css['dropdownMenu']}>
+                <Input
+                    placeholder={'Пошук'}
```

Comment:
No need for brackets
```suggestion
                    placeholder="Пошук"
```